hw/ip/aon_timer/dv/cov/aon_timer_unr_manually_excluded_excl.el

```diff
@@ -0,0 +1,186 @@
+CHECKSUM: "1125143680 1408670975"
+INSTANCE: tb.dut.u_reg.u_wkup_ctrl_cdc
+ANNOTATION: "src_ack is only 1 cycle pulse, and is an acknowledgment to a request from a read/write.
+src_busy_q feeds of the request, so by the time src_ack is high src_busy_q should also be high.
+See assertion SrcAckBusyChk_A for more details"
+Condition 3 "86646938" "(src_busy_q && src_ack) 1 -1" (1 "01")
+CHECKSUM: "1125143680 1408670975"
+INSTANCE: tb.dut.u_reg.u_wkup_thold_hi_cdc
+ANNOTATION: "src_ack is only 1 cycle pulse, and is an acknowledgment to a request from a read/write.
+src_busy_q feeds of the request, so by the time src_ack is high src_busy_q should also be high.
+See assertion SrcAckBusyChk_A for more details"
+Condition 3 "86646938" "(src_busy_q && src_ack) 1 -1" (1 "01")
+CHECKSUM: "1125143680 1408670975"
+INSTANCE: tb.dut.u_reg.u_wkup_thold_lo_cdc
+ANNOTATION: "src_ack is only 1 cycle pulse, and is an acknowledgment to a request from a read/write.
+src_busy_q feeds of the request, so by the time src_ack is high src_busy_q should also be high.
+See assertion SrcAckBusyChk_A for more details"
+Condition 3 "86646938" "(src_busy_q && src_ack) 1 -1" (1 "01")
+CHECKSUM: "1125143680 1408670975"
+INSTANCE: tb.dut.u_reg.u_wdog_bark_thold_cdc
+ANNOTATION: "src_ack is only 1 cycle pulse, and is an acknowledgment to a request from a read/write.
+src_busy_q feeds of the request, so by the time src_ack is high src_busy_q should also be high.
+See assertion SrcAckBusyChk_A for more details"
+Condition 3 "86646938" "(src_busy_q && src_ack) 1 -1" (1 "01")
+CHECKSUM: "1125143680 1408670975"
+INSTANCE: tb.dut.u_reg.u_wdog_bite_thold_cdc
+ANNOTATION: "src_ack is only 1 cycle pulse, and is an acknowledgment to a request from a read/write.
+src_busy_q feeds of the request, so by the time src_ack is high src_busy_q should also be high.
+See assertion SrcAckBusyChk_A for more details"
+Condition 3 "86646938" "(src_busy_q && src_ack) 1 -1" (1 "01")
+CHECKSUM: "1125143680 1408670975"
+INSTANCE: tb.dut.u_reg.u_wdog_ctrl_cdc
+ANNOTATION: "src_ack is only 1 cycle pulse, and is an acknowledgment to a request from a read/write.
+src_busy_q feeds of the request, so by the time src_ack is high src_busy_q should also be high.
+See assertion SrcAckBusyChk_A for more details"
+Condition 3 "86646938" "(src_busy_q && src_ack) 1 -1" (1 "01")
+CHECKSUM: "2815520955 4109606122"
+INSTANCE: tb.dut.u_reg.u_wkup_count_hi_cdc.u_arb
+ANNOTATION: "It can't be hit because dst_update_ack won't be set prior to dst_update_req"
+Condition 3 "3080886878" "(gen_wr_req.dst_update_req && gen_wr_req.dst_update_ack) 1 -1" (1 "01")
+ANNOTATION: "It would need dst_req_i=1, dst_req_q=1, busy=1. However, that's not possible.
+
+The FSM stays in IDLE until there's a request, and  then stays in StWait 1-cycle. During StWait, dst_req_q is set to 0x1, but we are also transitioning back to StIdle where busy will be set to 0x0 making the missing condition coverage unhittable.
```

Comment:
I think we can see this happen?

- Suppose we are sitting in `StIdle` and nothing is going on.
- At cycle 0, suppose that `dst_req_i` is high.
- `dst_req_q` will get a nonblocking assignment of `1'b1`.
- Also (because `dst_req_i` is high) `dst_req` will be high too.
- As a result, `state_d` will be `StWait`.

- On cycle 1, `state_q` will be `StWait`
- As a result, `busy` will be true.
- Also, `dst_req_q` will be 1.

So we'll see a hit unless `dst_req_i` has dropped.

Is that correct? (And is it possible for this to happen?)